applications/matter_bridge/src/ble_providers/ble_onoff_light_data_provider.cpp

```diff
@@ -32,11 +32,12 @@ uint8_t BleOnOffLightDataProvider::GattNotifyCallback(bt_conn *conn, bt_gatt_sub
 	VerifyOrExit(length == sizeof(mOnOff), );
 	VerifyOrExit(provider, );
 
-	/* TODO: Implement invoking command through the binding for OnOff light switch or update state of generic
-	 * switch. */
+	/* Save data received in the notification. */
+	memcpy(&provider->mOnOff, data, length);
```

Comment:
I'm not sure we should modify `mOnOff` here, as this is attribute from completely different cluster, unless we want to sort of bind the switch with light functionality and make button to control LED. I think we should just create a separate member called `mSwitchCurrentPosition` or sth.